src/Exceptions/PasswordDoesMatchException.php

```diff
@@ -0,0 +1,12 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Blumilk\Meetup\Core\Exceptions;
+
+use Exception;
+
+class PasswordDoesMatchException extends Exception
+{
+    protected $message = "Password cannot be the same as old password";
```

Comment:
Maybe exception should be renamed to be related with the message? E.g. `PasswordIsTheSameAsOldException` or something else.